docs-chef-io/content/server/v14/config_rb_server_optional_settings.md

```diff
@@ -1492,6 +1492,103 @@ not occur if you increment the version of a cookbook with every cookbook update,
 
 This configuration file has the following settings for `opensearch`:
 
+`opensearch['enable']`
+
+: Enable a service. Default value: `true`.
+
+`opensearch['dir']`
+
+: The working directory. The default value is the recommended value. Default value: `/var/opt/opscode/opensearch`
```

Comment:
```suggestion
: The working directory. Default value (recommended): `/var/opt/opscode/opensearch`
```
I'm not exactly sure how this should be worded, but maybe something like the above.  Hopefully the docs folks will have a suggestion.